stdlib/2and3/mailcap.pyi

```diff
@@ -0,0 +1,8 @@
+
+from typing import Sequence, Dict, List, Union, Tuple
+
+_Cap = Dict[str, Union[str, int]]
+_CapDict = Dict[str, List[_Cap]]
+
+def findmatch(caps: _CapDict, MIMEtype: str, key: str = ..., filename: str = ..., plist: Sequence[str] = ...) -> Tuple[str, _Cap]: ...
```

Comment:
Both elements of the return tuple should be Optional because it returns (None, None) if there is no match.

Also, the `_CapDict` dictionary should perhaps be replaced with a more general type such as Mapping, because it doesn't look like findmatch() requires a Dict.

Reply:
I wasn't sure about CapDict because on one hand I want to match the `caps` param and the return of `getcaps`, but `getcaps` definitely returns a dict while `caps` accepts any mapping to a sequence it looks like.